pkgs/servers/nitter/default.nix

```diff
@@ -0,0 +1,130 @@
+{ lib
```

Comment:
this looks like a generated file, can you indicate how to regenerate it or even better write an update script ?
https://nixos.org/manual/nixpkgs/stable/#special-variables

Reply:
This is hand-crafted (and yes it took a bit of time even with a good editor). I'm definitely motivated to write a nimble2nix script as this would make further update easier, but I need to solve these problems:
- find how to use nimble to resolve and output "locked" versions
- find how to know where is the code in each repository. Sometimes it's at the root, sometimes in `src/` folder.

I believe nimble2nix is out of the scope of this PR. As I use Nitter daily I should keep this file up to date and I will have motivation to write nimble2nix in the long term.

Reply:
> I believe nimble2nix is out of the scope of this PR. 

I agree.